operations/iscsi_sbps/Node_Personalization_Verification.md

```diff
@@ -1,242 +1,13 @@
-# iSCSI SBPS Worker Node Personalization
+# Node Personalization Verification
 
-The iSCSI SBPS solution requires worker nodes to be configured as iSCSI targets (servers) with necessary
-provisioning, configuration, and enablement of required components. This is done using
-[CFS](../../glossary.md#configuration-framework-service-cfs)-based Ansible plays which do the following things:
+Worker node personalization installs the SBPS Marshal agent, adds `kubernetes` labels for the worker NCNs intended to be configured as iSCSI targets, it also removes the labels from workers that are not designated to be iSCSI targets, creates DNS
+SRV and A records, etc. See [Managing Selective Node Personalization](Managing_selective_node_personalization.md)
+for details on how worker NCNs are designated to be enabled as iSCSI targets.
 
-* Provision iSCSI targets and LIO services.
-* Present the LIO network service on the HSN and [NMN](../../glossary.md#node-management-network-nmn)
-  IP networks (TCP port 3260 by default), via iSCSI Portals.
-* Enable SBPS Marshal Agent by installing the agent RPM and starting the respective `systemd` service (`sbps-marshal`).
-* Enable Spire for authentication used for [IMS](../../glossary.md#image-management-service-ims) and
-  [S3](../../glossary.md#simple-storage-service-s3) access.
-* Create DNS records in order to discover iSCSI targets from iSCSI initiators/clients (passed as boot parameter).
-* Mount S3 images (`boot-images` bucket) read-only with new, dedicated S3 user (`ISCSI-SBPS`)
-* Apply Kubernetes label (`iscsi=sbps`), to be consumed by the Goss test suit and LIO Metrics, for identification of
-  nodes on which these have to run.
+(`ncn-w#`) Run these checks on worker nodes that are enabled as an iSCSI target only in order to verify that it is
```

Comment:
This introductory text should provide guidance on which of these checks should be considered routine health checks vs those that are something else (troubleshooting steps? More in-depth checks? Manual steps that are also covered by automated Goss tests?)